browser/src/control/Control.Menubar.js

```diff
@@ -310,8 +310,7 @@ L.Control.Menubar = L.Control.extend({
 				{name: _('Latest Updates'), id: 'latestupdates', type: 'action', iosapp: false},
 				{name: _('Send Feedback'), id: 'feedback', type: 'action'},
 				{name: _('About'), id: 'about', type: 'action'}]
-			},
-			{name: _('Last modification'), id: 'last-mod', type: 'action', tablet: false}
```

Comment:
Some integrators and users are avoiding the use of statusbar (by hiding it be that to save vertical space, because they might not use those features or simply for personal taste) and thus I think it would be cool to instead of removing this altogether for everyone to, make it an option within the xml.